src/ShortcutNav.js

```diff
@@ -0,0 +1,93 @@
+import React, { useState, useEffect } from "react";
+import shortcutNavStyles from "./shortcutNavStyles.module.css";
+
+function ShortcutNav(props) {
```

Comment:
great job on the shortcut nav but also check out this article :https://stackabuse.com/how-to-scroll-to-top-in-react-with-a-button-component/

Reply:
also, you can checkout a library called react-outer-hash-link that can abstract that logic.